src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1029,6 +1029,7 @@ void cbmc_parse_optionst::help()
     " --json-ui                    use JSON-formatted output\n"
     HELP_GOTO_TRACE
     " --verbosity #                verbosity level\n"
+    HELP_TIMESTAMP
```

Comment:
Please add to goto-analyzer, goto-diff as well.

Reply:
+1 that would be lovely.

Reply:
This is done